guides/common/modules/con_configuring-dns-dhcp-and-tftp-additional-resources.adoc

```diff
@@ -0,0 +1,7 @@
+[id="configuring-dns-dhcp-and-tftp-additional-resources_{context}"]
+= Additional resources
+
+ifndef::foreman-deb,orcharhino[]
+* For information about configuring DNS, DHCP, and TFTP externally, see xref:configuring-external-services[].
```

Comment:
optional
```suggestion
* To learn about configuring DNS, DHCP, and TFTP externally, see xref:configuring-external-services[].
```

Reply:
This would be shorter... but then again, I think throughout the docs set, "For information about" is the standard way of introducing additional resources. So let's stick with this wording.

Reply:
Usually it is "For more information", "for information" just sounds to me... weird :D but it is non-blocking